app/services/fact_parser.rb

```diff
@@ -140,7 +140,7 @@ def find_interface_by_name(host_name)
     resolver = Resolv::DNS.new
     resolver.timeouts = PRIMARY_INTERFACE_RESOLVE_TIMEOUTS
     interfaces.detect do |int, values|
-      if (ip = values[:ipaddress]).present?
+      if (ip = values[:ipaddress] || values[:ipaddress6]).present?
```

Comment:
Instead of using either one of the IPs, should it try to resolve either?

I can imagine someone has some NAT IP for IPv4 but public IPv6 that is resolvable.